doc/manual/faq.rst

```diff
@@ -71,6 +72,27 @@ Currently, it is not possible to reach satellites through ``artiq_coremgmt confi
 
 Don't worry about individually flashing idle or startup kernels. If your idle or startup kernel contains subkernels, it will automatically compile as a ``.tar``, which you only need to flash to the master.
 
+fix unreliable DRTIO master-satellite links?
+--------------------------------------------
+
+Inconsistent DRTIO connections, especially with odd or absent errors in the core logs, are often a symptom of overheating either in the master or satellite boards. Check the core device fans for failure or defects. Try pulling the boards out of their crates for better circulation to see if that fixes the problem. In the long term, consider adding further cooling fans, preferably rack-mounted in the crate itself.
```

Comment:
Rack mounted fans are not in the crate itself, but rather to be installed below it.

Reply:
Rack-mounted _with_ the crate... ?